src/esportsbot/lib/stringTyping.py

```diff
@@ -32,6 +32,26 @@ def strIsRoleMention(mention: str) -> bool:
     return mention.endswith(">") and mention.startswith("<@&") and strIsInt(mention[3:-1])
 
 
+def strIsUserMention(mention: str) -> bool:
+    """Decide whether the given string is a discord user mention, being <@USERID> where USERID is an integer discord user id.
+
+    :param str mention: The string to check
+    :return: True if mention matches the formatting of a discord user mention, False otherwise
+    :rtype: bool
+    """
+    return mention.endswith(">") and mention.startswith("<@") and strIsInt(mention[2:-1])
```

Comment:
Might be better to slice the string and compare rather than using endswith and startswith.

Reply:
I'd copied the existing string - ultimately, a regex would be faster and more accurate if we wrote it well.

Reply:
ie `<@!?[0-9]+>`